js/index.js

```diff
@@ -1,2 +1,3 @@
 export { default as PaymentRequest } from './PaymentRequest';
 export { PKPaymentButton } from './PKPaymentButton';
+export { Intents } from './Intents';
```

Comment:
Nice, I like it. Maybe better to do an export default in `Intents.js` and then `export { default as Intents }` here. Saves you embedding everything inside a single object